src/lineAnalysis.js

```diff
@@ -0,0 +1,76 @@
+import * as cellValue from '../src/cellValue';
+import * as gameHelper from '../src/components/game/gameHelper';
+
+export function isThereAWinner(grid) {
+  let winningResults = {winnerFound:false, winningMove:null};
+  winningLines(grid).forEach(function(line) {
+    const [a, b, c] = line;
```

Comment:
Better than the hardcoding before, but I'd like to see this extensible to any size board!